dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.h

```diff
@@ -210,6 +219,13 @@ class OpenDDS_Rtps_Udp_Export RtpsUdpDataLink : public DataLink {
 
   } multi_buff_;
 
+  struct Response {
```

Comment:
AnnotatedSubmessage? MetaMessage? DirectedSubmessage? (I like this one less, since we could extend the metadata to include more than locators / INFO_DST)